twitch_api.py

```diff
@@ -2,62 +2,25 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from datetime import datetime
-from typing import Optional
+from itertools import islice
 import os
 from dotenv import load_dotenv
 import twitch
 
 load_dotenv()
 
 
-class TwitchStreamInterface(ABC):
-    """
-    The twitch stream interface
-    """
-
-    @abstractmethod
-    def is_live(self) -> bool:
-        """
-        Checks if the user passed is currently live
-
-        Args
-            username (str): The twitch username / user_login
-
-        Returns
-            bool: True if live, else false
-        """
-
-
 @dataclass
-class TwitchStream(TwitchStreamInterface):
+class TwitchStream:
     """
     The twitch stream implementation
     """
-    id: int
-    user_id: int
-    user_login: str
     user_name: str
-    game_id: int
     game_name: str
-    live: str
     title: str
     viewer_count: int
```

Comment:
Probably don't need viewer_count anymore since I removed it from the announcement, when a streamer goes live they'll likely have very few viewers so let's not display it anymore.